R/dictionaries.R

```diff
@@ -202,7 +214,7 @@ setMethod("as.list",
 #' head(dfm(data_corpus_inaugural, dictionary = mfdict))}
 #' @export
 dictionary <- function(..., file = NULL, format = NULL, 
-                       concatenator = " ", 
+                       concatenator = "_", 
```

Comment:
Why set this as a default if we always convert it to whitespace later? The purpose is not to set it, but to tell it the "input concatenator", so that we know what to convert to whitespace. We should consider renaming this to `separator`.